queries.js

```diff
@@ -0,0 +1,63 @@
+const Pool = require('pg').Pool
+const pool = new Pool({
```

Comment:
we usually leave spaces between imports and constants

Reply:
in this lines? or you mean in the index.js where I'm declaring db functions